spec/rubocop/cop/rspec/repeated_block_body_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::RepeatedBlockBody do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for repeated describe body' do
+    expect_offense(<<-RUBY)
+      describe 'doing x' do
+      ^^^^^^^^^^^^^^^^^^^^^ Don't repeat block body
+        it { cool_predicate_method }
+      end
+
+      describe 'doing y' do
+      ^^^^^^^^^^^^^^^^^^^^^ Don't repeat block body
```

Comment:
Sometimes it might be hard to figure out where it is repeated. Would it be possible to point at the duplication?

Reply:
@pirj I am thinking about pointing to the line of code, WDYT?

Reply:
That would be ideal.